addons/live-comments/profiles.js

```diff
@@ -0,0 +1,139 @@
+export default async function ({ addon, msg }) {
+  let path = window.location.pathname;
+  let username = path.split("/")[2];
+
+  let eButton = document.createElement("button");
+
+  function addButton() {
+    let commentsText = Array.from(document.querySelectorAll("h4")).find((el) => el.innerText == "Comments");
+
+    function setButtonStyles() {
+      eButton.innerHTML = eButton.enabled ? "Live on" : "Live off";
```

Comment:
```suggestion
      eButton.innerText = eButton.enabled ? "Live on" : "Live off";
```
Ideally this would be consistent with the allow commenting toggle though.